nav2_costmap_2d/test/unit/keepout_filter_test.cpp

```diff
@@ -183,33 +123,36 @@ class TestNode : public ::testing::Test
 
 void TestNode::createMaps(unsigned char master_value, int8_t mask_value)
 {
-  // Make map and mask overlapping as follows:
+  // Make map and mask put as follows:
```

Comment:
Just curious, why change to this test? There's no non-overlapping area to test, or is that covered by another test?

Reply:
Initially, when I've checked code coverage I've found some areas of overlapping that did not checked. For example, it is bottom-left area (0,0,2,2) which you can find on the `keepout_filter_test.jpg` picture. Therefore, I've decided to move `mask_` in order to check all possible cases.